src/components/FeedPage.vue

```diff
@@ -51,6 +62,8 @@ export default {
             this.loadMoreVideos();
             this.updateWatched(this.videos);
         });
+
+        this.selectedFilter = this.$route.query.filter ?? this.getPreferenceString("feedFilter") ?? "all";
```

Comment:
`this.$route.query.filter` is technically not needed, since `this.getPreferenceString("feedFilter")` checks for the parameter in the query.

Reply:
Adding that was intended to also allow customizing the filter with a query parameter since using queries instead of preferences seems to be quite popular among the users of Piped

Reply:
The getPreference* methods check the query string first, then followed by localstorage. Isn't that the same thing?

Reply:
Oh, you're right, I misunderstood you before :)